### ISSUE TITLE
Quick question: What do the values in moduleNormalize correspond to

### ISSUE THREAD

**Issue by repers** [HyeongminLEE/AdaCoF-pytorch#21]
Hi there, I've seen this used to normalize the inputs, are these the mean and standard deviation values from Vimeo90K or do they represent something else? It would be appreciated if you could let me know about this

```
def moduleNormalize(frame):
    return torch.cat([(frame[:, 0:1, :, :] - 0.4631), (frame[:, 1:2, :, :] - 0.4352), (frame[:, 2:3, :, :] - 0.3990)], 1)
```

Thanks
